scipy-1.0/rebuttal.tex

```diff
@@ -0,0 +1,164 @@
+% based on template:
+% https://tex.stackexchange.com/questions/15532/looking-for-cover-letter-template
+
+\documentclass[10pt,stdletter,dateno]{newlfm}
+
+% to adjust vertical spacing properties of
+% newlfm class see: https://tex.stackexchange.com/a/65533/43006
+%\unprtop{0mm}
+\topmarginsize{0mm}
+\topmarginskip{0in}
+\headermarginsize{0in}
+\headermarginskip{0in}
+\MinHead{0in}
+%\dateskipafter{0pt}
+%\dateskipbefore{0pt}
+\sigskipbefore{40pt}
+%\sigskipafter{0pt}
+\closeskipbefore{1pt}
+%\closeskipafter{0pt}
+%\addrfromskipbefore{0pt}
+\addrfromskipafter{0in}
+\addrtoskipbefore{0in}
+%\addrtoskipafter{0pt}
+\newlfmP{Headlinewd=0pt,Footlinewd=0pt}
+\newlfmP{sigsize=10pt}
+\bottommarginskip{10pt}
+\MinFoot{0pt}
+\leftmarginsize{0.95in}
+\rightmarginsize{0.95in}
+
+\usepackage{kpfonts}
+\usepackage{hyperref}
+\usepackage{url}
+
+\hypersetup{%
+  linkbordercolor=blue,% hyperlink borders will be blue
+  pdfborderstyle={/S/U/W 1}% border style will be underline of width 1pt
+}
+\widowpenalty=1000
+\clubpenalty=1000
+
+\namefrom{Ralf Gommers,
+          Matt Haberland,
+          and Tyler Reddy}
+\addrfrom{%
+    \today\\[10pt]
+    Ralf Gommers\\
+    Quansight Labs\\
+    The Netherlands\\
+    \texttt{ralf.gommers@gmail.com}\\
+    %phone number\\
+    \\
+    Matt Haberland\\
+    BioResource and Agricultural Engineering\\
+    California Polytechnic State University\\
+    San Luis Obispo, CA 93407, USA\\
+    \texttt{mhaberla@calpoly.edu}\\
+    %phone number\\
+    \\
+    Tyler Reddy\\
+    CCS-7 Applied Computer Science Group\\
+    Los Alamos National Laboratory\\
+    Los Alamos, NM 87545, USA\\
+    \texttt{treddy@lanl.gov}\\
+    %phone number\\
+}
+
+\addrto{%
+Dr. Rita Strack\\
+Nature Methods Editorial Office\\
+One New York Plaza Suite 4500\\
+New York, NY 10004, USA}
+
+\greetto{Dear Dr. Strack,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We appreciate the careful review and helpful suggestions from you and the reviewers. Each comment is quoted and addressed below; we also link the relevant GitHub issues in which we discussed and implemented each suggestion. As requested, the corresponding changes are underlined in the revised manuscript.
+
+\begin{quote}
```

Comment:
It may be helpful to associate the quotes with their respective reviewers (R1/2), but we have sufficiently few comments that we may be ok without that. Just want to make sure we make it as easy as possible for the Editors to confirm everything is addressed.

Reply:
I think it's pretty easy. They are in the order they were provided (so sorted by reviewer), except for the combined one about GPU/distributed computing.

Reply:
OK I mentioned that they are in the order of the original review letter.